### PR TITLE
Adding DOM 

### PR DESCRIPTION
**Pull request by iamsjunaid**
**In this milestone I have added the following in my portfolio website**

- ✅DOM events 
- ✅JavaScript

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+});
+
+closeMenu.addEventListener('click', () => {
+  mobileMenu.classList.toggle('active');
+});
+
+mobileLinks.forEach((link) => {
+  link.addEventListener('click', () => {
+    mobileMenu.classList.toggle('active');
+  });
```
Comment (Shedrack-Sunday) [inline index.js:17]: - [ ] Your mobile hamburger displays on the desktop. Kindly fix this by displaying it as none on the desktop screen size 👍  
- This gives a more profestionl feel, as it shows you pay attention to detail 👍 

## View :
image

Comment (Shedrack-Sunday) [inline index.js:17]: - [ ] I can't see the hamburger button on the mobile version. Theses are the functionalities needed:
On mobile the hamburger menu button is visible.

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.
- When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.

## View:
image

## Instructions:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @iamsjunaid         👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights
- proper use of linters ✔️ 
- Fair  git commits 👍 
  
# Required Changes ♻️

- There's a bit of styling and interactivity issue  that needs to be an adjustment based on project requirements 

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.js (hunk 1 above)
Comment (iamsjunaid) [reply index.js:17]: I have pushed some code after submitting for review. Can you check the latest code? or should I request for another review.

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @iamsjunaid 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - Menu working as expected. ✔️ 
 - Nice UI design. ✔️ 
 - No linter errors. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (Shedrack-Sunday) [reply index.js:17]: Sorry for the late response. I can see it's approved. Good job 👏

**Pull request merged**